src/main/java/frc/utils/pid/PIDUtils.java

```diff
@@ -0,0 +1,100 @@
+package frc.utils.pid;
+
+import com.revrobotics.SparkPIDController;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+/**
+ * Contains helper methods for simplifying and abstracting code related to PID controllers.
+ *
+ * @see frc.utils.pid.PIDConfig
+ * @see frc.utils.pid.PIDParam
+ */
+public final class PIDUtils {
+
+  private PIDUtils() {}
+
+  /**
+   * Maps a {@link frc.utils.pid.PIDParam} to the corresponding setter method of a {@link
+   * com.revrobotics.SparkPIDController}.
+   *
+   * <p>Example: {@code getCorrespondingSetMethod(controller, PIDParam.P)} will return a {@link
+   * java.util.function.Consumer} {@code (value) -> controller.setP(value)}
+   *
+   * @param controller The PID controller that the mapping will target
+   * @param pidParam The {@link frc.utils.pid.PIDParam} that the mapping will reference
+   */
+  public static Consumer<Double> getCorrespondingSetMethod(
```

Comment:
Check if setting to latest shuffleboard entry from the corresponding key.

Reply:
@Irishumanoid Please see your school email ASAP.